src/uct/cuda/cuda_ipc/cuda_ipc_ep.h

```diff
@@ -1,35 +1,59 @@
 /**
-* Copyright (C) Mellanox Technologies Ltd. 2018.  ALL RIGHTS RESERVED.
-* See file LICENSE for terms.
-*/
+ * Copyright (C) Mellanox Technologies Ltd. 2018.  ALL RIGHTS RESERVED.
+ * Copyright (c) 2018, NVIDIA CORPORATION. All rights reserved.
+ * See file LICENSE for terms.
+ */
 
 #ifndef UCT_CUDA_IPC_EP_H
 #define UCT_CUDA_IPC_EP_H
 
 #include <uct/api/uct.h>
 #include <uct/base/uct_iface.h>
 #include <ucs/type/class.h>
+#include "cuda_ipc_md.h"
+
+
+#define UCT_CUDA_IPC_HASH_SIZE 256
+#define UCT_CUDA_IPC_PUT 0
+#define UCT_CUDA_IPC_GET 1
```

Comment:
move  UCT_CUDA_IPC_PUT/ UCT_CUDA_IPC_GET to cuda_ipc_ep.c as these used only in that file